compendium/test_bundles/TestBundleDSTOI9/src/ServiceImpl.cpp

```diff
@@ -1,15 +1,26 @@
 #include "ServiceImpl.hpp"
 
+#include <iostream>
+#include <exception>
+
 namespace sample {
-  void ServiceComponent9::Activate(const std::shared_ptr<ComponentContext>&)
-  {
-    activated = true;
-    throw std::runtime_error("Exception thrown from user code");
-  };
-  void ServiceComponent9::Deactivate(const std::shared_ptr<ComponentContext>&)
-  {
-    deactivated = true;
-    throw std::runtime_error("Exception thrown from user code");
-  };
-  ServiceComponent9::~ServiceComponent9() {};
+
+ServiceComponent9::ServiceComponent9()
+  : test::LifeCycleValidation()
+{
+  throw std::exception {};
```

Comment:
you should be able to test for this condition without DS by creating a ```ServiceFactory``` derived class which throws an exception when its constructed.